htlcswitch/link_test.go

```diff
@@ -1108,6 +1110,31 @@ func TestChannelLinkMultiHopUnknownNextHop(t *testing.T) {
 		t.Fatal("the bandwidth of carol channel link which handles " +
 			"bob->carol channel should be the same")
 	}
+
+	// Load the forwarding packages for Bob's incoming link. The payment
```

Comment:
May want to double check here that this test isn't flaky due to the garbage collection goroutine periodically removing the set of forwarding packages (5 seconds it tests when I last checked). We can modify its ticker (if this is the case) to be the mock ticker so it actually never attempts to clean up.

Reply:
timeout bumped to 15s